website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
 description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
 ---
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
```

Comment:
```suggestion
dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the [Discovery API](/docs/dbt-cloud-apis/discovery-api). 
```

Reply:
Hiya @mirnawong1 

For some reason, I couldn't add a suggestion to this.
For the sentence, **For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project** flowed nicely and read really well.

Kind Regards